### ISSUE TITLE
Humans have too much oxygen

### ISSUE THREAD

**Issue by bluewoods64** [studio-minus/ppg-bugs-and-requests#1231]
**Feature description**
Reduction of human oxygen capacity. 

**Feature purpose**
Humans currently take over a minute to drown which I think is a bit ridiculous. The average breath a human has is around 30 seconds, and I know it takes longer than that to die IRL from drowning, but they wouldn't have oxygen by then, while in game it shows they have atleast some oxygen until they die.

**Additional context**
Maybe a better drowning mechanism could be a part of this? I know it should be a different request but it kind of ties to this. The human never goes unconscious when they drown, which always happens IRL.

**Comment by bluewoods64** [comment]
i just know somebody will point out something about this they dont like and bash me

**Comment by Drogo-man** [comment]
The human can hold Thier breath for longer then you might think.

**Comment by bluewoods64** [comment]
> The human can hold Thier breath for longer then you might think.

if a human is panicking i dont expect their breath to last long

**Comment by Drogo-man** [comment]
> if a human is panicking I don't expect their breath to last long

I mean, yeah. It would be nice though if they instead went unconscious after enough time without oxygen.
But still, you can survive for pretty long when you're panicking, you just would fall unconscious and just lose oxygen a lot slower than normal.

**Comment by bluewoods64** [comment]
> > if a human is panicking I don't expect their breath to last long
> 
> I mean, yeah. It would be nice though if they instead went unconscious after enough time without oxygen. But still, you can survive for pretty long when you're panicking, you just would fall unconscious and just lose oxygen a lot slower than normal.

yeah i kinda mentioned that in the additional context. i doubt zooi will even see this though. i've made requests before with 8+ thumbs up and it seems zooi didnt even see it

**Comment by mestiez** [comment]
i see everything

**Comment by bluewoods64** [comment]
> i see everything. thumbs up just dont matter

so #937 is a no? it would be very helpful

**Comment by Cboyjet4** [comment]
I'd rather have a refactor of the whole oxygen/circulatory system instead, current one is very heavily based on arbitrary values instead of dynamic factors (proper blood pressure, proper blood oxygen)
